src/components/sq-input/sq-input.component.html

```diff
@@ -1,13 +1,16 @@
 <div class="wrapper-all-inside-input {{ customClass }}">
   <label
     class="display-flex"
-    *ngIf="label?.length || tooltipMessage"
+    *ngIf="label?.length || tooltipMessage || labelTemplate"
     [ngClass]="{
       readonly: readonly
     }"
     [for]="id"
   >
-    <div *ngIf="label" [ngStyle]="{ 'color': labelColor }" [innerHtml]="label | universalSafe"></div>
+    <div *ngIf="label && !labelTemplate" [ngStyle]="{ 'color': labelColor }" [innerHtml]="label | universalSafe"></div>
+    <div *ngIf="labelTemplate && !label">

```

Comment:
Acho que não precisa do not label aqui. Se sem querer passar os dois n vai mostrar nada, a gente pode priorizar o template

Reply:
fixed